engine/dlib/src/dlib/dns.cpp

```diff
@@ -291,6 +309,7 @@ namespace dmDNS
         {
             Channel* dns_channel = (Channel*) channel;
             ares_destroy(dns_channel->m_Handle);
+            delete dns_channel->m_Servers;
```

Comment:
technically, strdup uses "malloc" so you should match with "free"

Reply:
Thanks. Updated.